frontend/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM call-home-shared
+
+WORKDIR /app/frontend
+
+COPY package.json .
+COPY package-lock.json .
+
+RUN npm run init
+RUN npm install
+
+COPY tsconfig.json .
+COPY src/ src/
+COPY public/ public/
+
+RUN npm start
```

Comment:
What does this dockerfile do exactly? It looks like this just for dev because this uses the react-scripts build. If it is just for dev, perhaps this should be Dockerfile.dev and the prod one would be Dockerfile.

I think the typical approach for most SPAs in docker is to build the bundle and then serve it with nginx or something similar. I don't think we care enough to serve with nginx, but we can copy the bundle out of the prod image.

Reply:
just for dev, will rename